src/ansys/mechanical/core/embedding/ui.py

```diff
@@ -22,7 +22,10 @@
 """Run Mechanical UI from Python."""
 
 from pathlib import Path
-from subprocess import Popen
+
+# Subprocess is needed to launch the GUI and clean up the process on close.
```

Comment:
@klmcadams why does bandit complain about subprocess??

Reply:
https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b404-import-subprocess

B404 - importing subprocess is considered a "risky" condition because of security implications. Damages can be performed on your system - although to be fair you can also do them with pure Python...

Anyway, we should justify the import and usage of subprocess, and make sure that all inputs that are passed to subprocess are controlled by us (avoid user injection).